src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -390,6 +390,54 @@ private static bool CheckConstraintsSingleType(TypeSymbol type, CheckConstraints
         }
 
         public static bool CheckConstraints(
```

Comment:
A long tuple is a tuple with more than 7 elements. The underlying type used to represent such tuples involves nesting a `ValueTuple` in the `rest` field of a `ValueTuple'8`.

---
In reply to: [91728289](https://github.com/dotnet/roslyn/pull/15641#discussion_r91728289) [](ancestors = 91728289)